br/pkg/storage/s3.go

```diff
@@ -57,9 +59,7 @@ const (
 	// the maximum number of byte to read for seek.
 	maxSkipOffsetByRead = 1 << 16 // 64KB
 
-	// TODO make this configurable, 5 mb is a good minimum size but on low latency/high bandwidth network you can go a lot bigger
-	hardcodedS3ChunkSize = 5 * 1024 * 1024
```

Comment:
@Leavrth PTAL, we changed the default value

Reply:
By mistake.